pkg/api/types.go

```diff
@@ -149,6 +149,12 @@ type ObjectMeta struct {
 	// from the list. If the deletionTimestamp of the object is non-nil, entries
 	// in this list can only be removed.
 	Finalizers []string `json:"finalizers,omitempty"`
+
+	// The name of the cluster which the object belongs to.
+	// This is used to distinguish resources with same name and namespace in different clusters.
+	// This field is not set anywhere right now and apiserver is going to ignore it if set in create or update request.
+	// Cannot be updated.
```

Comment:
We can delete this "Cannot be updated" line, since we say that it is ignored in create and update requests in the previous line.